www-client/qutebrowser/qutebrowser-9999.ebuild

```diff
@@ -57,7 +57,7 @@ python_test() {
 
 python_install_all() {
 	doman doc/${PN}.1
-	dodoc {CHANGELOG,CONTRIBUTING,FAQ,README}.asciidoc
+	for DocFile in {doc/,./}*.asciidoc; do dodoc $DocFile; done
```

Comment:
This would omit `README.asciidoc` which still resides in top-level dir.

Reply:
Actually, note the `./`; it selects all `.asciidoc` files in the `./` dir :D 

The commit 426f970 is much nicer :) Thanks for the feedback.